### ISSUE TITLE
Self Report page axe violation

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#2721]
"Ensure links are distinguished from surrounding text in a way that does not rely on color":
![image](https://github.com/user-attachments/assets/0018eec4-d770-4d4f-b976-49d96691437f)

"Landmarks should have a unique role or role/label/title (i.e. accessible name) combination":
![image](https://github.com/user-attachments/assets/6dc6ba6b-5ddf-4e22-bce2-a8eab68cc034)

**Comment by rmmayo** [comment]
:+1:
